Sources/NIOCertificateReloading/TimedCertificateReloader.swift

```diff
@@ -100,6 +100,21 @@ import Foundation
 /// not being recognized or not matching the configured one; not being able to verify a certificate's signature against the given
 /// private key; etc), then that attempt will be aborted but the service will keep on trying at the configured interval.
 /// The last-valid certificate-key pair (if any) will be returned as the ``sslContextConfigurationOverride``.
+///
+/// Optionally, you may also observe any reloads by specifying the ``TimedCertificateReloader/Configuration/onCertificateLoaded`` parameter.
+/// This will notify you whenever a new certificate is loaded, and you will given an instance of ``TimedCertificateReloader/LoadedCertificateChainAndKeyPairDiff``.
```

Comment:
```suggestion
/// This will notify you whenever a new certificate is loaded, and you will be given an instance of ``TimedCertificateReloader/LoadedCertificateChainAndKeyPairDiff``.
```